packages/init/src/util.js

```diff
@@ -3,6 +3,31 @@ import os from "node:os";
 import path from "node:path";
 import { spawnSync } from "node:child_process";
 
+// synchronous copy with simple retry/backoff for Windows transient EBUSY
+function copyFileSyncWithRetry(src, dest, attempts = 5, baseDelay = 50) {
+  let lastErr;
+
+  for (let i = 0; i < attempts; i++) {
+    try {
+      return fs.copyFileSync(src, dest);
+    } catch (err) {
+      lastErr = err;
+      if ((err.code === 'EBUSY' || err.code === 'EPERM' || err.code === 'EACCES') && i < attempts - 1) {
+        const delay = baseDelay * Math.pow(2, i);
+        const waitUntil = Date.now() + delay + Math.floor(Math.random() * baseDelay);
+        while (Date.now() < waitUntil) {
+          // busy wait — this is synchronous intentionally
```

Comment:
This will consume a lot of unnecessary CPU and drain the battery if run on a user’s device. I understand it’s being run in a synchronous context, but I’m looking for ways we can avoid this. There’s a thread-sleep module that seems to have found a clever trick with node’s execSync function
https://github.com/ForbesLindesay/thread-sleep/blob/master/index.js

another thing that *maybe* could help is this fancy Atomics thing I found. Although I admit I haven’t finished looking into it so maybe not. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/pause